### PR TITLE
Updated users doc for 1.3 version including metrics-server information

### PR DESCRIPTION
**Pull request by Smorenoz**
Signed-off-by: Salomon Moreno <msalomon@vmware.com>

Which other branches should this be merged with (if any)?

### CODE REVIEW

**Review by obabativavmware — APPROVED** [review]

File: tap-gui/plugins/runtime-resource-visibility.hbs.md [hunk 1]
```diff
@@ -12,8 +12,18 @@ Do one of the following actions to access the Runtime Resources Visibility plug-
...
+## <a id="metrics-server"></a> If you have a metrics server
+
+By default, the Kubernetes API will be ignoring the metrics server on your cluster, if you want to access
+this information, you will need to go to your cluster section on the `app-config.yaml` and make sure
+the `skipMetricsLookup` attribute is set to `false`. 
+
+> *Caution*
+>If you set this value to `false` but do you not possess a metrics server running, you will get an error
+>as the follow:
+>![Screenshot of the RRV plugin error message due to metrics](images/metrics-server-error.png)
```
Comment (menehune23) [inline tap-gui/plugins/runtime-resource-visibility.hbs.md:26]: I'd recommend against a screenshot here, for a couple reasons:
- Not necessary for a note like this. A description of the error is sufficient (though not verbatim, due to below bullet)
- It is tedious to need to update things like this if the error message changes

File: tap-gui/plugins/runtime-resource-visibility.hbs.md [hunk 2]
```diff
@@ -12,8 +12,18 @@ Do one of the following actions to access the Runtime Resources Visibility plug-
 - [Install the Tanzu Application Platform Full or View profile](../../install-intro.md)
 - [Install Tanzu Application Platform without using a profile](../../install-intro.md) and then
   install [Tanzu Application Platform GUI separately](../install-tap-gui.md)
-- Disable realtime metrics for clusters without a metrics server by setting
-`skipMetricsLookup` to `true` for that cluster in `app.config.yaml`
+- Make sure to review the section [If you have a metrics server](#a-idmetrics-servera-if-you-have-a-metrics-server)
```
Comment (menehune23) [inline tap-gui/plugins/runtime-resource-visibility.hbs.md:15]: Suggest: Review the section ...

File: tap-gui/plugins/runtime-resource-visibility.hbs.md [hunk 3]
```diff
@@ -12,8 +12,18 @@ Do one of the following actions to access the Runtime Resources Visibility plug-
...
+
+## <a id="metrics-server"></a> If you have a metrics server
+
+By default, the Kubernetes API will be ignoring the metrics server on your cluster, if you want to access
```
Comment (menehune23) [inline tap-gui/plugins/runtime-resource-visibility.hbs.md:19]: Suggesting some slight edits:

By default, the Kubernetes API will not attempt to use any metrics servers on your clusters. If you want to access
metrics information for a cluster, set `skipMetricsLookup` to `false` for that cluster in your `app-config.yaml`'s `kubernetes` section.

File: tap-gui/plugins/runtime-resource-visibility.hbs.md (hunk 1 above)
Comment (menehune23) [reply tap-gui/plugins/runtime-resource-visibility.hbs.md:26]: Suggest:

If you enable metrics for a cluster but do not have a metrics server running on it, users of the Tanzu Application Platform web interface will see an error notifying them that there was a problem connecting to the backend.

**Comment by RichardJJG** [comment]
@Smorenoz, @menehune23, because this has been merged, I'll make @menehune23's changes myself directly on the branch. Thanks for your work, guys.

File: tap-gui/plugins/runtime-resource-visibility.hbs.md (hunk 1 above)
Comment (RichardJJG) [reply tap-gui/plugins/runtime-resource-visibility.hbs.md:26]: I'll apply these changes, thanks.

File: tap-gui/plugins/runtime-resource-visibility.hbs.md (hunk 2 above)
Comment (RichardJJG) [reply tap-gui/plugins/runtime-resource-visibility.hbs.md:15]: I'll make this change.

File: tap-gui/plugins/runtime-resource-visibility.hbs.md (hunk 3 above)
Comment (RichardJJG) [reply tap-gui/plugins/runtime-resource-visibility.hbs.md:19]: I'll apply this change, thanks.

**Pull request merged**